### ISSUE TITLE
 ERR wrong number of arguments for 'unlink' command

### ISSUE THREAD

**Issue by leoseccia** [caching-tools/next-shared-cache#430]
**Brief Description of the Bug**
I am using the [Redis handler]( https://caching-tools.github.io/next-shared-cache/redis) example. 

When I invoke `revalidatePath` or `revalidateTag`, I always get the error: 
```
revalidateTag failed for <tag_identifier> [ErrorReply: ERR wrong number of arguments for 'unlink' command]
```

**Severity**
Major

**Frequency of Occurrence**
Always

**Steps to Reproduce**
1. Setup Next.js app to use [Redis handler]( https://caching-tools.github.io/next-shared-cache/redis) 
2. run Redis stack: `docker run -d --name redis-stack -p 6379:6379 -p 8001:8001 redis/redis-stack:latest`
3. Browse to a path
4. `revalidateTag` on that path (or `revalidatePath`)

**Expected vs. Actual Behavior**
The cache should be cleared on the next page visit. 
Instead the cache does not clear and I get the error: `revalidateTag failed for <tag_identifier> [ErrorReply: ERR wrong number of arguments for 'unlink' command]`.

**Environment:**

- OS: Windows + Ubuntu on WSL
- Node.js version: v20.5.0
- `@neshca/cache-handler` version: v1.0.5
- `next` version: v14.1.0

**Comment by mbeaudru** [comment]
I think this commit https://github.com/caching-tools/next-shared-cache/commit/e92658a introduced the bug

Example on how to unlink found here: https://github.com/redis/redis-om-node/pull/89/files#diff-147a4b9e517bc0355f0911b17497cdb0d0fbeb27d8d588eb071aba8d3dddf0efR155

**Comment by better-salmon** [comment]
@leoseccia and @mbeaudru, thank you for this issue. It was fixed in version 1.0.8.
